src/main/scala/exercises/06_Collections.scala

```diff
@@ -139,23 +153,34 @@ object Collections {
    * As usual, various ways exist: pattern matching, folding, ...
    */
   def maxElementInList(l: List[Int]): Int = {
-    error("fix me")
+    l.foldLeft(0)((m: Int, n: Int) => if (m > n) m else n)
   }
 
   /**
    * Calculate the sum of the equally position elements
    * of the two list
    */
   def sumOfTwo(l1: List[Int], l2: List[Int]): List[Int] = {
-    error("fix me")
+    if (l1.isEmpty) {
+      l2
+    } else if (l2.isEmpty) {
+      l1
+    } else {
+      l1.zip(l2).map({case (m,n) => m + n}) 
```

Comment:
I didn't realize you could have a lone `case` as an expression without a `match` keyword.  Huh!
This makes your `map` argument much prettier than mine:
```scala
(tup : (Int, Int)) => tup._1 + tup._2
```